exporter/prometheusexporter/README.md

```diff
@@ -23,6 +23,8 @@ The following settings can be optionally configured:
 - `metric_expiration` (default = `5m`): defines how long metrics are exposed without updates
 - `resource_to_telemetry_conversion`
   - `enabled` (default = false): If `enabled` is `true`, all the resource attributes will be converted to metric labels by default.
+- `enable_open_metrics`
+  - `enabled` (default = false): If `enabled` is `true`, metrics will be exported using the OpenMetrics format. Exemplars are only exported in the OpenMetrics format.
```

Comment:
This is not how it works in the code, there is no `enabled_open_metrics` section, but rather `enable_open_metrics` is itself a boolean. You need to change either of the two

Reply:
yes thats right; will update the readme to match the functionality.